models/user_repository.go

```diff
@@ -0,0 +1,37 @@
+package models
```

Comment:
You can convert it in the REST Controller yes.. That's what the REST Controller is there for. Converting from externally exposed apis to internal apis.